test.py

```diff
@@ -299,7 +302,7 @@ def report_subcause(e):
             def report_subcause(e):
                 print('  with error {e}\n'.format(e=e), file=file)
             report_error(e, e, report_subcause=report_subcause)
-        return (path, boost_args + exec_args, success, file.getvalue())
+        return (path, boost_args + exec_args, type, success, file.getvalue())
```

Comment:
is `type` used?
It's unused on line 321
and then it's retrieved on line 344, but not used.

Reply:
Indeed, this `type` in line 305 is not used.
Type is only used on line 338 (`if r[2] != 'boost'`).
Changed `type` to `_`.